src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -18,8 +27,20 @@
      *      - DAY - current day of month;
      * In any other case throw DateTimeException.
      **/
+
     public String todayDate(DateTimePart datePart) {
-        return "Today";
+        switch (datePart) {
+            case FULL:
+                return LocalDate.now().toString();
+            case YEAR:
+                return String.valueOf(LocalDate.now().getYear());
+            case MONTH:
+                return LocalDate.now().getMonth().toString();
+            case DAY:
+                return String.valueOf(LocalDate.now().getDayOfMonth());
```

Comment:
Avoid calling LocalDate.now multiple times in one method. Let’s create a variable for this purpose.